### ISSUE TITLE
Upgrade dependencies please

### ISSUE THREAD

**Issue by secondl1ght** [babakfp/svelte-outclick#13]
The latest version of SvelteKit requires Vite v5. I am not sure if `svelte-outclick` will break with an incorrect peer dependency but it produces warnings in your package manager:

```bash
warning "svelte-outclick > @sveltejs/kit@1.30.3" has incorrect peer dependency "vite@^4.0.0".
warning "svelte-outclick > @sveltejs/kit > @sveltejs/vite-plugin-svelte@2.5.3" has incorrect peer dependency "vite@^4.0.0".
warning "svelte-outclick > @sveltejs/kit > @sveltejs/vite-plugin-svelte > @sveltejs/vite-plugin-svelte-inspector@1.0.4" has incorrect peer dependency "vite@^4.0.0".
```

Would you be able to update deps to the latest versions please?

**Comment by secondl1ght** [comment]
It also causes errors when running `svelte-check` which I think many people will be using in their integration pipelines:

```bash
error "svelte-outclick#@sveltejs/kit#vite@^4.0.0" doesn't satisfy found match of "vite@5.0.9"
error "svelte-outclick#@sveltejs/kit#@sveltejs/vite-plugin-svelte#vite@^4.0.0" doesn't satisfy found match of "vite@5.0.9"
error "svelte-outclick#@sveltejs/vite-plugin-svelte#@sveltejs/vite-plugin-svelte-inspector#vite@^4.0.0" doesn't satisfy found match of "vite@5.0.9"
```

So currently upgrading to SvelteKit v2 is not possible with `svelte-outclick` as a dependency.

**Comment by babakfp** [comment]
Hi.
Thank you for reporting this 👍.
I'll be working on it right away and release a new version shortly.
Thanks again.

**Comment by babakfp** [comment]
Fixed in `3.7.0`.
Thanks.
